### ISSUE TITLE
Replace event-stream with something secure and supported

### ISSUE THREAD

**Issue by jeremywadsack** [IBM/audit-ci#315]
`audit-ci` depends on dominictarr/event-stream which was added in #97. The author stopped supporting the module in 2016 and there are suggestions of other packages here: https://github.com/dominictarr/event-stream/issues/102#issuecomment-248754581. In 2018 the author archived the repo, months before it was added to this package as a dependency. 

This means that `audit-ci` (that we use for security scanning) includes a dependency on a library that isn't currently maintained and is likely insecure. In fact, `event-stream` has had an issue with a supply chain attack (https://github.com/dominictarr/event-stream/issues/115) that while resolved raised concerns (and poor etiquette) from the community.

**Comment by quinnturner** [comment]
With all that being said, here are some of the alternatives that I found:


https://github.com/uhop/stream-json

https://github.com/creationix/jsonparse

https://github.com/dscape/clarinet

https://github.com/jgranstrom/zipson

https://github.com/Faleij/json-stream-stringify

In my opinion, stream-json seemed like the most reasonable alternative based on the criteria I set out. I have yet to identify the performance difference.

**Comment by jeremywadsack** [comment]
Thanks for helping me understand this better. I thought I responded last week but don't see that here in the thread, so apologies for the late reply. 

Agreed that the current state of the security is more important than historical. I highlighted the issue with `event-stream` because it appears to be the last known action on that repo.

I guess I don't have anything better to offer and you've given me something to consider.
